charts/kyverno/ci/extraContainers-values.yaml

```diff
@@ -0,0 +1,9 @@
+extraContainers:
+  - name: myapp-container
+    image: busybox
+    command: ['sh', '-c', 'echo Hello && sleep 3600']
+
+extraInitContainers:
+  - name: init-container
+    image: busybox
+    command: ['sh', '-c', 'echo Hello']
```

Comment:
The helm chart lint is complaining that this file has no newline at the end.  The Github Action output isn't as useful for some reason but I get this locally:

```
charts/kyverno/ci/extraContainers-values.yaml
  9:40      error    no new line character at the end of file  (new-line-at-end-of-file)
```

Reply:
ah sorry i noticed this as well and fixed it already 😄